### ISSUE TITLE
[Legitimate Site Blocked] metamaap.com

### ISSUE THREAD

**Issue by shoorsamridhi** [MetaMask/eth-phishing-detect#11579]
https://metamaap.com/

**Comment by shoorsamridhi** [comment]
Dear Team,

In case of any queries or authentication proofs kindly reach out to office@metaversecouncil.io

This domain - metamaap.com - is the collaborated property of Metaverse Council of India - metaversecouncil.io . We request immediate removal of our domain metamaap from the blacklist.

Thanks & regards,
Samridhi Shoor,
Director - Meta Asset research center, 
Metaverse Council

**Comment by shoorsamridhi** [comment]
https://www.metaversecouncil.io/  

This is our primary domain for more information about us. Metamaap is a community project initiated under SR foundation to support the various cultures of India,. We are extremely distraught due to this occurrence. Therefore, We also request you to share any legal license with us to share your credibility to flag domains in this way without approval from the owners.

**Comment by shoorsamridhi** [comment]
Please respond ASAP

**Comment by AlexHerman1** [comment]
Hi there,

Metamaap shows as "under construction", and was blocked by fuzzy logic for its visual similarity to "metamask", which makes it a phishing risk. 

Only browsers with metamask installed and active will receive this warning.

If you have completed the project you are building you can open a new issue to request review for allowlisting.

Thank you.

**Comment by shoorsamridhi** [comment]
Metamask or not, If your fuzzy logic blocks our website just because of it's visual similarity to "metamask", because you/ your team think of it as  a phishing risk. 

Trust me, There will be legal situation with penalty of over 10 million for your team. I would suggest you take down your logic immediately & do not close this request like this.

**Comment by shoorsamridhi** [comment]
Dear ALEX!

I understand that you are trying to make internet/blockchain a secure place for everyone but flagging accounts like this without any responsibility of the damages that another legit entity/person may incur isn't the right way to go about things. 

This act/initiative of yours is liable to damages & considered unlawful as a form of digital harassment & intentional obstruction to other person's entity business/work. It is against a person's Digital Privacy.

Kindly, make amends at your earliest. This has been a great inconvenience!!  We are supposed to share this domain with our investors.
